dotnet/SimpleSauce/SauceOptions.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace SimpleSauce
+{
+    public class SauceOptions
+    {
+        public SauceOptions()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this redundant constructor.](https://app.codacy.com/app/SauceLabs/simple_sauce/pullRequest?prid=4471446)